service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersApprovedEventHandler.java

```diff
@@ -87,18 +95,44 @@ public void sendNotificationToAdminAndLA(final DraftOrdersApproved event) {
 
         String adminEmail = courtService.getCourtEmail(caseData);
 
-        final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
-            .caseData(caseData)
-            .build();
-
-        final Collection<String> localAuthorityEmails = localAuthorityRecipients.getRecipients(recipientsRequest);
-
         notificationService.sendEmail(
             JUDGE_APPROVES_DRAFT_ORDERS,
             adminEmail,
             content,
             caseData.getId()
         );
+    }
+
+    @Async
+    @EventListener
+    public void sendNotificationToLA(final DraftOrdersApproved event) {
+        CaseData caseData = event.getCaseData();
+        List<HearingOrder> approvedOrders = new ArrayList<>();
+        approvedOrders.addAll(event.getApprovedOrders());
+        approvedOrders.addAll(event.getApprovedConfidentialOrders().stream()
```

Comment:
Create a method`getConfidentialOrderToBeSentToLA` like `getConfidentialOrderToBeSentToCafcass`?

Reply:
it is only used here, and unlike Cafcass, `OrderCollectionLA` is the only field LA can access